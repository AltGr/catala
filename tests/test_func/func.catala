/*
declaration scope S:
  context f content int depends on int
  context x content int
  context b content bool

scope S:
  def f of x ? (x >= x) |= x + x
  def f of x ? not b |= x * x

  def b = false

  def x = f of 3
*/
