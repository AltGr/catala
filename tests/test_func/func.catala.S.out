[ERROR] There is a conflict between multiple rules for assigning the same variable.
[ERROR] 
[ERROR] This value is available because the justification of its definition is true:
[ERROR]   --> test_func/func.catala
[ERROR]    | 
[ERROR] 14 |   def f of x [ (x >= x) ] := x + x
[ERROR]    |                 ^^^^^^
[ERROR] 
[ERROR] This value is available because the justification of its definition is true:
[ERROR]   --> test_func/func.catala
[ERROR]    | 
[ERROR] 15 |   def f of x [ not b ] := x * x
[ERROR]    |                ^^^^^
